components/Main.vue

```diff
@@ -89,6 +112,8 @@ import SectionsAboutMe from "./ui/Sections/AboutMe";
 import SectionsFeature from "./ui/Sections/Feature";
 import SectionsOurTeam from "./ui/Sections/OurTeam";
 import SectionsParagraphWithImage from "./ui/Sections/ParagraphWithImage";
+// Pricing
+import PricingsSimple from "./ui//Pricings/Simple";
```

Comment:
Pls, rename the new section to pricing instead of pricings

Reply:
ok